demisto_sdk/commands/format/format_constants.py

```diff
@@ -53,3 +53,5 @@
     FileType.GENERIC_DEFINITION,
 ]
 OLD_FILE_TYPES = [FileType.LAYOUT.value, FileType.OLD_CLASSIFIER.value]
+
+SKIP_FORMATTING_DIRS = [".venv"]
```

Comment:
Why is ".venv" the only problem? we have many files we do not want to format inside directories ( denistomock, CSP, etc...) what am I missing here?

Reply:
these are usually handled via the excluded_files parameter

from format_manager:
```python
from demisto_sdk.commands.lint.commands_builder import excluded_files
...
...
        for file in files:
            file_path = str(Path(file))
            file_type = find_type(file_path, clear_cache=clear_cache)

            current_excluded_files = excluded_files[:]
            dirname = os.path.dirname(file_path)
            if dirname.endswith("CommonServerPython"):
                current_excluded_files.remove("CommonServerPython.py")
            if os.path.basename(file_path) in current_excluded_files:
                continue
            if any(test_dir in str(dirname) for test_dir in TESTS_AND_DOC_DIRECTORIES):
                continue
```
where 
```python
excluded_files = [
    "CommonServerPython.py",
    "demistomock.py",
    "CommonServerUserPython.py",
    "conftest.py",
    ".venv",
]
```
but since the basename of the .venv/* files is not .venv this does not hold. 
since the excluded_files are also used in other places I did not want to change this. 
and if you have a file with a name of .venv, you probably do not need to format it.

In my implementation, we get rid of these files before it even gets to this, also the .venv directory is discarded no matter where your .venv dir is located (in PackName/ or in Integrations/IntegrationName or anywhere inside the path of the -i option)